src/PIL/ImageDraw.py

```diff
@@ -181,6 +181,15 @@ def ellipse(self, xy: Coords, fill=None, outline=None, width=1) -> None:
         if ink is not None and ink != fill and width != 0:
             self.draw.draw_ellipse(xy, ink, 0, width)
 
+    def circle(self, xy: Coords, radius, fill=None, outline=None, width=1) -> None:
+        """Draw a circle given center coordinates and a radius."""
+        ink, fill = self._getink(outline, fill)
+        ellipse_xy = (xy[0] - radius, xy[1] - radius, xy[0] + radius, xy[1] + radius)
+        if fill is not None:
+            self.draw.draw_ellipse(ellipse_xy, fill, 1)
+        if ink is not None and ink != fill and width != 0:
+            self.draw.draw_ellipse(ellipse_xy, ink, 0, width)
```

Comment:
These four lines could be replaced by just `self.ellipse(ellipse_xy, fill, outline, width)`, and you can then remove the first line of code as well.